packages/python-packages/protocol-stub-generator/protocol/protocol_models.py

```diff
@@ -466,6 +469,7 @@ def from_yaml(cls, yaml_data: Dict[str, Any], op_group_num, op_num, namespace):
             ]["language"]["default"]["description"]
 
         return cls(
+            operation_group = yaml_data["operationGroups"][op_group_num]["language"]["default"]["name"],
```

Comment:
nit: no spaces between the assignment

```suggestion
            operation_group=yaml_data["operationGroups"][op_group_num]["language"]["default"]["name"],
```